src/tracer.js

```diff
@@ -54,7 +53,7 @@ export default class Tracer {
         this._reporter = reporter;
         this._sampler = sampler;
         this._logger = logger || new NullLogger();
-        this._host = Utils.createEndpoint(serviceName, myLocalIp(), port);
+        this._host = Utils.createEndpoint(serviceName, Utils.myIp(), port);
```

Comment:
if we are switching to the new data model, there's no need to e `endpoint` and for `port`. Simply add IP as a tracer tag.